<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QMainWindow" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajustes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../dist/launcher/src/feather/mainLogo.ico</normaloff>../../dist/launcher/src/feather/mainLogo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	 background-color: rgb(151, 151, 151)
}

QLabel {
	color: rgb(255, 255, 255);
	
	font: 75 10pt &quot;Yu Gothic UI&quot;;

}

QLineEdit {
	border:  2px solid rgb(98, 98, 98);
	border-radius: 15px;
	color: rgb(255, 255, 255);
	background-color: rgb(151, 151, 151)
}
QLineEdit:hover{
	border: 2px solid rgb(48,50,62);
}

QLineEdit:focus {

	border : 2px solid rgb(48,50,62)

}


QPushButton {

	border-radius: 10px;
	background-color: rgb(84, 84, 84);
	color:rgb(255, 255, 255)

}

QPushButton:hover {
	 background-color: rgb(0, 85,124);
     color: rgb(255, 255, 255)
}

QPushButton:checked {
                                    background-color: rgb(0, 85,124);
                                    color: rgb(255, 255, 255)
                                }

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="title">
     <string>Aspecto</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Tema </string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTema">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonAplicar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCancelar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNewUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuevo Usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonResetPass">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>110</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Contrase√±as</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancelar</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
